index.html

```diff
@@ -59,6 +59,216 @@ <h3>LET'S CONNECT</h3>
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+      </ul>
+      <button>
+          See project
+      </button>
+    </div>
+  </div>
+  <!-- about section -->
+<div class="abtme">
+    <h3>
+        About Myself
+      </h3>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <h4>LET'S CONNECT</h4>
     
+    <ul class="abtmeicons">
+              <li>
+                <a href="https://twitter.com/MjononoRonald"
+                  ><img src="./Images/icons/twitter.svg" alt="twittericon"
+                /></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/ronald-mjonono-86365988/"
+                  ><img src="./Images/icons/linkedin.svg" alt="linkedicon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"><img src="./Images/icons/myspace.svg" alt="" /></a>
+              </li>
+              <li>
+                <a href="https://github.com/mj-e-boc"
+                  ><img src="./Images/icons/github.svg" alt="githubicon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./Images/icons/evangelist.svg" alt="evangelist"
+                /></a>
+              </li>
+            </ul>
+            <button class="getresume">
+                Get my resume
+            </button>
+            <div class="lans">
+                <h5>
+                    Languages
+                </h5>
+                
+                <img src="./Images/icons/unionpointingdown.svg" alt="union">
+                
+            </div>
+            <div class="lansIcons">
+                <div>
+                    <img src="./Images/icons/javascript.svg" alt="jsicon">
+                </div>
+                
+                <p>JavaScript</p>
+            </div>
+            <div class="lansIcons">
+                <div>
+                    <img src="./Images/icons/html.svg" alt="htmlicon">
+                </div>
+           <p>HTML</p>     
+                
+            </div>
+            <div class="lansIcons">
+                <div><img src="./Images/icons/css.svg" alt="cssIcon"></div>
+                
+                <p>
+                    CSS
+                </p>
+            </div>
+            <div class="lans names">
+                <h5>
+                   Frameworks
+                </h5>
+                
+                <img src="./Images/icons/rightarrow.svg" alt="union">
+                
+            </div>
+            <div class="lans names">
+                <h5>
+                   Skills
+                </h5>
+                
+                <img src="./Images/icons/rightarrow.svg" alt="union">
+                
+            </div>
```

Comment:
- [ ] You did well in creating the technology categories for your portfolio :+1:. However, to ensure greater accuracy, it's recommended that the technology categories ("Languages," "Frameworks," and "Skills") be organized in an unordered list (`ul`) with each category listed as a list item (`li`). Similarly, the technologies listed under each category (such as "JavaScript," "HTML," and "CSS") should be organized as a sublist, also using an unordered list (`ul`) with each technology item listed as a list item (`li`). This helps to enhance the structure and readability of the categories and their respective technologies.